cluster/juju/layers/kubernetes-worker/actions/pause

```diff
@@ -2,6 +2,18 @@
 
 set -ex
 
+FORCE_LOCAL=$(action-get force)
+
 kubectl --kubeconfig=/srv/kubernetes/config cordon $(hostname)
-kubectl --kubeconfig=/srv/kubernetes/config drain $(hostname) --force
+
+if [ ${FORCE_LOCAL} == True ]; then
```

Comment:
Should `True` be lower-case? If I remember right, action params are based on [JSON schema](http://json-schema.org/latest/json-schema-core.html), which says this about booleans:

> A "true" or "false" value, from the JSON "true" or "false" productions

This would apply to the default values in actions.yaml, too.

I don't have a problem with it if it works in testing, though; I'm just worried it won't!